react/QuotaAlert/index.jsx

```diff
@@ -65,6 +65,7 @@ const QuotaAlert = ({ onClose }) => {
   if (isMobileApp()) {
     return <QuotaModalAlert onClose={onClose} />
   }
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
let's add TODO to investigate why this :pray:

Reply:
✅